css/css-text/overflow-wrap/overflow-wrap-min-content-size-001.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Test: overflow-wrap: break-word and intrinsic sizing</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#overflow-wrap-property">
+<meta name="flags" content="">
+<link rel="match" href="reference/overflow-wrap-min-content-size-001-ref.html">
+<meta name="assert" content="Soft wrap opportunities introduced by overflow-wrap:break-word **are** considered when calculating min-content intrinsic sizes.">
+<style>
+table {
+  overflow-wrap: break-word;
+  max-width: 0;
```

Comment:
Is the max-width really relevant for this test ? If the purpose of the test is to evaluate how soft-breaking opportunities account for the min-content width, wouldn't be enough to use width: min-content ?